### DISCUSSION TITLE
Hand IK, Aiming, ADS?

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#270] [General]
I'm playing around with the control rig, definitely still fresh in this area, but was curious if Refactored has anything already built in to simplify using hand IK positioning on weapons / guns based on sockets?

Aiming of the gun the character is holding based on look at?

And, definitely curious if there's anything built out to assist in setting up ADS based on sockets?

**Comment by xkamronx** [comment]
Looks like it's probably possible with the Control Rig, but still learning in the way of figuring that out.

**Reply by TheBreadMan66** [reply]
Hello, have you figured this out?
